### PR TITLE
Use new perltidy wrapper tools/tidyall

### PR DESCRIPTION
**Pull request by perlpunk**
Issue: https://progress.opensuse.org/issues/155188

### CODE REVIEW

**Review by kalikiana — APPROVED** [review]

**Comment by perlpunk** [comment]
I now added `shebang = perl` to `.tidyallrc`
https://metacpan.org/dist/Code-TidyAll/view/bin/tidyall#shebang

**Comment by perlpunk** [comment]
I think I now found a configuration that works for all.
https://metacpan.org/dist/Code-TidyAll/view/bin/tidyall#[class]-or-[class-description]
We can have more than one section. In one we specify the files with a file extension, in the other we specify the scripts and use the shebang option.

File: external/os-autoinst-common/.github/workflows/base-commit-message-checker.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Check subject line length
         uses: gsactions/commit-message-checker@v2
         with:
-          pattern: '^.{1,72}(\n|$)'
+          pattern: '^(Revert "?)?.{1,72}(\n|$)'
```
Comment (b10n1k) [inline external/os-autoinst-common/.github/workflows/base-commit-message-checker.yml:29]: Does this count `Revert`+72chars right? which exceeds the limit of 72 IIUC! And it makes the error msg incorrect. Just making sure that this is what we want.

File: .tidyallrc [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
-select = {tools,lib,t}/**/*.{pl,pm,t} script/*
+; We split into modules and scripts because the scripts have no file
+; extension, so we detect them by their shebang
+
+[PerlTidy modules]
+select = {lib,t}/**/*.{pl,pm,t}
 argv = --profile=$ROOT/.perltidyrc
+
+[PerlTidy scripts]
+select = tools/* script/*
```
Comment (b10n1k) [inline .tidyallrc:9]: s/tools/*/tools/**/*/? 
or doesnt it matter?

File: external/os-autoinst-common/.github/workflows/base-commit-message-checker.yml (hunk 1 above)
Comment (perlpunk) [reply external/os-autoinst-common/.github/workflows/base-commit-message-checker.yml:29]: This is an update of the external repo. If it's wrong, then it needs to be fixed upstream, so please file an issue there

File: .tidyallrc (hunk 2 above)
Comment (perlpunk) [reply .tidyallrc:9]: We don't have subdirectories in there

**Review by baierjan — APPROVED** [review]

**Review by okurz — APPROVED** [review]

**Comment by perlpunk** [comment]
We don't get codecov patch info, but that's not relevant here, so merging

**Pull request merged**